### PR TITLE
add run python example test: inception and pytorch

### PR DESCRIPTION
**Pull request by qiuxin2012**
add run python example test: inception and pytorch

### CODE REVIEW

**Comment by qiuxin2012** [comment]
jenkins passed

**Comment by hkvision** [comment]
Also need to add to run-example-tests-pip.sh @qiuxin2012

File: pyzoo/zoo/examples/run-example-tests.sh [hunk 1]
```diff
@@ -202,6 +202,21 @@ ${ANALYTICS_ZOO_HOME}/bin/spark-submit-python-with-zoo.sh \
...
+   ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/inception/inception.py \
+   --maxIteration 20 \
+   -b 8 \
+   -f hdfs://172.168.2.181:9000/imagenet-small
+
+echo "start example test for pytorch SimpleTrainingExample"
+${ANALYTICS_ZOO_HOME}/bin/spark-submit-python-with-zoo.sh \
+   --master local[1] \
+   --driver-memory 5g \
+   ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/pytorch/train/SimpleTrainingExample.py
```
Comment (hkvision) [inline pyzoo/zoo/examples/run-example-tests.sh]: Where are Lenet, resnet finetune and inference?

File: pyzoo/zoo/examples/run-example-tests.sh [hunk 2]
```diff
@@ -202,6 +202,21 @@ ${ANALYTICS_ZOO_HOME}/bin/spark-submit-python-with-zoo.sh \
...
 time5=$((now-start))
 
+echo "start example test for inceptionv1 training"
+${ANALYTICS_ZOO_HOME}/bin/spark-submit-python-with-zoo.sh \
+   --master local[4] \
+   --driver-memory 10g \
+   ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/inception/inception.py \
+   --maxIteration 20 \
+   -b 8 \
+   -f hdfs://172.168.2.181:9000/imagenet-small
```
Comment (hkvision) [inline pyzoo/zoo/examples/run-example-tests.sh]: Shall we expose hdfs here?

Comment (qiuxin2012) [reply pyzoo/zoo/examples/run-example-tests.sh]: follow ` -f hdfs://172.168.2.181:9000/kaggle/train_100 \` in line 92

**Pull request merged**